blocks/article-body-block/_index.scss

```diff
@@ -0,0 +1,148 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-article-body {
+	> * {
+		@include scss.block-components("article-body-children");
+		@include scss.block-properties("article-body-children");
+	}
+
+	a {
+		@include scss.block-properties("article-body-link");
+	}
+
+	&__h2 {
+		@include scss.block-properties("article-body-h2");
+	}
+
+	&__h3 {
+		@include scss.block-properties("article-body-h3");
+	}
+
+	&__h4 {
+		@include scss.block-properties("article-body-h4");
+	}
+
+	&__h5 {
+		@include scss.block-properties("article-body-h5");
+	}
+
+	&__h6 {
+		@include scss.block-properties("article-body-h6");
+	}
```

Comment:
I suppose this is the same question as above, but is there a reason to create all these tokens?

Reply:
Composer gives us Heading tags, and this is the way to support customisations to those